apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -227,11 +227,14 @@ def _sanitize_markdown(mdtext):
 
 def _load_introduction(path):
     "Loads the introduction text from a Markdown file"
+    if not os.path.exists(path):
+        return None
+
     try:
         with open(path) as f:
             return _sanitize_markdown(f.read())
-    except:
-        return None
+    except Exception as err:
+        raise RuntimeError(f'Makrdown file "{path}" could not be loaded: {err}')
```

Comment:
hmm, in this case perhaps the whole try-except could be removed altogether?

because if the file does not exist, it's already returning `None`. In any other case it should actually probably fail with the exception, no?